src/sentry/models/scheduledjob.py

```diff
@@ -1,10 +1,29 @@
 from __future__ import absolute_import, print_function
 
-from django.db import models
+from django.db import models, transaction
 from django.utils import timezone
 from jsonfield import JSONField
 
-from sentry.db.models import Model, sane_repr
+from sentry.db.models import (BaseManager, Model, sane_repr)
+
+
+class ScheduledJobManager(BaseManager):
+    def schedule_job(self, payload, job):
```

Comment:
this ks kind of an odd format to use

i would reschedule ``schedule_job(name, payload, date_scheduled=Optional[datetime])``